presto-release/src/main/java/com/facebook/presto/release/tasks/GenerateReleaseNotesCommand.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.release.tasks;
+
+import com.facebook.airlift.bootstrap.Bootstrap;
+import com.facebook.airlift.bootstrap.LifeCycleManager;
+import com.facebook.airlift.log.Logger;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Injector;
+import com.google.inject.Module;
+import com.google.inject.TypeLiteral;
+import io.airlift.airline.Command;
+import io.airlift.airline.Option;
+
+import java.util.Optional;
+
+import static com.google.common.base.Throwables.throwIfUnchecked;
+
+@Command(name = "release-notes", description = "Presto release notes generator")
+public class GenerateReleaseNotesCommand
+        implements Runnable
+{
+    private static final Logger log = Logger.get(GenerateReleaseNotesCommand.class);
+
+    @Option(name = "github-user", title = "Github username", required = true)
```

Comment:
Is this the right convention for parameters names to not have - in front? I find it's useful for making it clear on the CLI the difference between the name of a parameter value (or sequence of values).

The convention I am used to is a single '-' in front of a single letter parameter name and '--' in front of the full word. And usually both would be available for each parameter.

Reply:
Fixed